.github/scripts/develocity_reports.py

```diff
@@ -0,0 +1,863 @@
+                container_result = details['container_result']
+                
+                print(f"\n{container_name}")
+                print("=" * len(container_name))
+                print(f"Quarantined for {details['days_quarantined']} days")
+                print(f"Container Failure Rate: {details['failure_rate']:.2%}")
+                print(f"Recent Failure Rate: {details['recent_failure_rate']:.2%}")
+                print("\nContainer Statistics:")
+                print(f"  Total Runs: {container_result.outcome_distribution.total}")
+                print(f"  Failed: {container_result.outcome_distribution.failed}")
+                print(f"  Flaky: {container_result.outcome_distribution.flaky}")
+                print(f"  Passed: {container_result.outcome_distribution.passed}")
+                
+                # Show container timeline
+                if container_result.timeline:
+                    print("\nContainer Recent Executions:")
+                    print("  Date/Time (UTC)      Outcome    Build ID")
+                    print("  " + "-" * 48)
+                    for entry in sorted(container_result.timeline, key=lambda x: x.timestamp)[-5:]:
+                        date_str = entry.timestamp.strftime('%Y-%m-%d %H:%M')
+                        print(f"  {date_str:<17} {entry.outcome:<10} {entry.build_id}")
+                
+                print("\nTest Cases (Last 7 Days):")
+                print("  " + "-" * 48)
+                
+                # Sort test cases by failure rate
+                sorted_cases = sorted(
+                    details['test_cases'],
+                    key=lambda x: (x.outcome_distribution.failed + x.outcome_distribution.flaky) / x.outcome_distribution.total if x.outcome_distribution.total > 0 else 0,
+                    reverse=True
+                )
+                
+                for test_case in sorted_cases:
+                    total_runs = test_case.outcome_distribution.total
+                    if total_runs > 0:
+                        failure_rate = (test_case.outcome_distribution.failed + test_case.outcome_distribution.flaky) / total_runs
+                        
+                        # Extract the method name from the full test case name
+                        method_name = test_case.name.split('.')[-1]
+                        
+                        print(f"\n  → {method_name}")
+                        print(f"    Failure Rate: {failure_rate:.2%}")
+                        print(f"    Runs: {total_runs:3d} | Failed: {test_case.outcome_distribution.failed:3d} | "
+                              f"Flaky: {test_case.outcome_distribution.flaky:3d} | "
+                              f"Passed: {test_case.outcome_distribution.passed:3d}")
+                        
+                        # Show test case timeline
+                        if test_case.timeline:
+                            print("\n    Recent Executions:")
```

Comment:
For "Recent" things, let's indicate how far back we're showing in the output.

Reply:
Added info about the runs.